### ISSUE TITLE
'No Cat file is selected' Error

### ISSUE THREAD

**Issue by meimei-dev** [rafaelperez/ViTMatte-for-Nuke#13]
Hi Rafael!
Thanks for making and distributing ViTMatte-for-Nuke to nuke community.
I have a few issues as I use this gizmo
I using Nuke 14.0v4 and GPU spec is RTX 2070 Super

1.
first of all, when I used local env, If I don't uncheck the 'Use GPU if available' button, it won't work.
I think there is a person in a similar situation to me in the issue below.

2.
second of all, when I used Render farm 'tractor', there is an Error message like
`ERROR: ViTMatte.Inference1: No Cat file is selected.` 
but In local env, vitmatte.cat is work's correctly. how does this happen?

that's all the issues I have.
I looked up Reddit and Foundry forums , but I couldn't get any relevant information.
so I'm asking you a question. 

Thank you again for distributing this gizmo.

from Pepo

**Comment by yousaf1438** [comment]
Yes, I am also facing the same issue.

**Comment by magnoborgo** [comment]
I had render farm / terminal render issues too. 
You *need* to add the directory with the .cat file to the Nuke plugin path on init.py, otherwise it can't find it, i.e something like:
```
nuke.pluginAddPath("/yoursystempathtodir/Cattery/vitmatte/")
```

**Comment by UnaiM** [comment]
Hi, we’re facing the same issue on Nuke 13.2v9. Here are the 2 code snippets I’ve run to confirm the CAT files are there and Nuke sees them:

```python
import pathlib

for path in nuke.pluginPath():
    path = pathlib.Path(path)
    result = tuple(path.glob('vitmatte_*.cat'))
    if result:
        print('\n'.join(str(x) for x in result))
```

```
c:\...\vitmatte_large.cat
c:\...\vitmatte_small.cat
```

```python
for size in ('small', 'large'):
    print(repr(nuke.tcl(f'plugins -all vitmatte_{size}.cat')))
```

```
'{c:\\...\\vitmatte_small.cat}'
'{c:\\...\\vitmatte_large.cat}'
```

(This one would have printed `None` if Nuke couldn’t find either.)

**Comment by magnoborgo** [comment]
Its very likely your path is not correct. it should not have "c:\" if its relative, unless your are omitting the path on your example.

**Comment by UnaiM** [comment]
Yes @magnoborgo, the paths are elided to respect NDAs, but basically it’s printing absolute paths— I don’t think there’s a way for either Python snippet to print anything relative… Can `nuke.pluginPath()` contain relative paths? If so, relative to what?
